src/getHumanAge.test.js

```diff
@@ -7,4 +7,20 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  it('should return correct value if catAge && dogAge < 15', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
```

Comment:
Better write each 'expect' separately, so that you check only one situation in one test